src/writer/proposal.ts

```diff
@@ -36,6 +37,42 @@ export const getProposalsCount = async (space, author) => {
   return await db.queryAsync(query, [space, author]);
 };
 
+async function validateSpace(originalSpace: any) {
+  const space = snapshot.utils.clone(originalSpace);
+
+  if (!space) {
+    return Promise.reject('unknown space');
+  }
+
+  if (space.hibernated) {
+    return Promise.reject('space hibernated');
+  }
```

Comment:
We should handle these inside `validateSpaceSettings`

Reply:
Those check are proposal oriented, as those condition should not fail when updating settings